embedded/memory.x

```diff
@@ -0,0 +1,34 @@
+MEMORY
```

Comment:
Is there not an easier way to test embedded stuff than four config files just for a test?

Reply:
Yes, like https://github.com/rust-bitcoin/bitcoin_hashes/pull/119#issuecomment-817112440

however, this way is suggested in the rust embedded book from the rust embedded team, so maybe more common for embedded devs?

Reply:
I think `.cargo/config` could be easy removed in favor of command parameters and `.github/workflows/embedded.yml` could be mixed with the other github actions. Would that be ok?

Reply:
I don't know that we need to squash the workflows files down, but if its possible to remove some of the config files it would be nice, especially given a few of them look like half-comments.

Reply:
#119 is updated without comments and fewer config files. Could you please review it there?

Reply:
superseded by #122